guides/client/js-interop.md

```diff
@@ -122,6 +122,29 @@ In the case of an `"element"` page loading event, the info will contain a
 `"target"` key containing the DOM element which triggered the page loading
 state.
 
+## Triggering phx events with JavaScript
+
+Often it is desirable to trigger an event on a DOM element without explicit
```

Comment:
```suggestion
Often it is desirable to trigger an event on a form element without explicit
```

Reply:
Technically the change event is on the input and not the form element, so this might add confusion.